osu.Game.Rulesets.Taiko/UI/TaikoPlayfield.cs

```diff
@@ -257,6 +282,20 @@ internal void OnNewResult(DrawableHitObject judgedObject, JudgementResult result
             }
         }
 
+        protected override void Update()
+        {
+            base.Update();
+
+            if (Time.Elapsed < 0)
+            {
+                foreach (DrawableHit taikoHit in drumRollHitContainer.Objects)
+                {
+                    taikoHit.RemoveProxiedContent();
```

Comment:
This isn't going to be sufficient. `UnproxyContent` just moves content from a container that has a proxy to a container that doesn't have a proxy, but the point is to remove the proxy itself from the `topLevelHitContainer` instead.

i.e. You need to remove the `Drawable` that was created by the previous call to `CreateProxiedContent()`.

Reply:
After the recent changes, this line and the function added is no longer required.

Reply:
Cool. Thanks for that smoogi! Done! :)